ui/src/components/Footer/index.tsx

```diff
@@ -18,30 +18,46 @@
  */
 
 import React from 'react';
+import { Link } from 'react-router-dom';
 import { Container } from 'react-bootstrap';
-import { Trans } from 'react-i18next';
+import { Trans, useTranslation } from 'react-i18next';
 
 import dayjs from 'dayjs';
 
 import { siteInfoStore } from '@/stores';
 
 const Index = () => {
+  const { t } = useTranslation('translation', { keyPrefix: 'footer' }); // Scoped translations for footer
   const fullYear = dayjs().format('YYYY');
   const siteName = siteInfoStore((state) => state.siteInfo.name);
   const cc = `${fullYear} ${siteName}`;
+
   return (
     <footer className="bg-light">
-      <Container className="py-3">
-        <p className="text-center mb-0 small text-secondary">
-          <Trans i18nKey="footer.build_on" values={{ cc }}>
-            Powered by
-            {/* eslint-disable-next-line react/jsx-no-target-blank */}
-            <a href="https://answer.apache.org" target="_blank">
+      <Container>
+        <p className="text-center mb-0 small">
+          {/* Link to Terms of Service with right margin */}
+          <Link to="/tos" className="me-2">
+            {t('label', { keyPrefix: 'admin.legal.terms_of_service' })}
+          </Link>
+
+          {/* Link to Privacy Policy with right margin for spacing */}
+          <Link to="/privacy">
+            {t('label', { keyPrefix: 'admin.legal.privacy_policy' })}
+          </Link>
+        </p>
+        <p className="text-center mb-0 small">
+          <Trans i18nKey="build_on" values={{ cc }}>
+            Powered by{' '}
```

Comment:
```suggestion
            Powered by
            {/* eslint-disable-next-line react/jsx-no-target-blank */}
```

Reply:
it will provide a space between "Power by" and "Apache Answer"

without {' '}:
![Screenshot from 2024-12-31 10-32-24](https://github.com/user-attachments/assets/b30bf932-6cf1-4d51-965f-87cf6c45f89c)


 with {' '}:
![Screenshot from 2024-12-31 10-32-12](https://github.com/user-attachments/assets/9f972098-6bbe-427c-aa9d-c7575f6b9cbf)

Reply:
But it fails to replace it when using the built file.
<img width="502" alt="image" src="https://github.com/user-attachments/assets/83d82dd6-92fe-439d-9c31-ab13c239893b" />

Reply:
I didnt get it. are u saying that if I bilt it with "make ui" it is not working.

Reply:
yep, 

```
make ui

go run cmd/answer/main.go run -C ./answer-data
```

Reply:
I  got no file error "/home/mindslayer001/Project/incubator-answer
config file path:  answer-data/conf/config.yaml
Answer is starting..........................
panic: stat answer-data/conf/config.yaml: no such file or directory"

Reply:
how about using old method of adding space 
`<a 
href="https://answer.apache.org" 
target="_blank" 
rel="noopener noreferrer">
&nbsp; Apache Answer
</a>`

Reply:
Why are you adding spaces?
The original is the correct result

```
Powered by <a href="https://answer.apache.org" target="_blank"> Apache Answer </a>- the open-source software that powers Q&amp;A communities.<br>Made with love © 2024 Local Answer.
```

Reply:
> how about using old method of adding space `<a href="https://answer.apache.org" target="_blank" rel="noopener noreferrer"> &nbsp; Apache Answer </a>`

No need to adjust like this, just keep it the same as before.
![image](https://github.com/user-attachments/assets/a0373ca8-b601-441d-82a3-dcafa757d99f)

Reply:
It is because when I am running `npm run lint` the space between `powered by` and  `Apache` in removed.
So for Standardization purpose I am trying to add space.

Reply:
okay, But you didn't solve the problem. https://github.com/apache/incubator-answer/pull/1217/files#r1899921473

Reply:
@sy-records @shuashuai I made the changes as per your feedback please check the pr